custom_components/tuya_local/devices/aktobis_wdh_310ek_dehumidifier.yaml

```diff
@@ -0,0 +1,175 @@
+name: Aktobis WDH-310EK dehumidifier
```

Comment:
Make this name generic, and move the brand name to the name under products below.